python/ray/serve/_private/http_proxy.py

```diff
@@ -32,7 +32,9 @@
 
 logger = logging.getLogger(SERVE_LOGGER_NAME)
 
-MAX_REPLICA_FAILURE_RETRIES = 10
+MAX_REPLICA_FAILURE_RETRIES = int(
```

Comment:
Can you add an assertion or a warning if this value is invalid (e.g. set to a non-positive integer)?